samples/system-test/spanner.test.js

```diff
@@ -774,16 +774,38 @@ describe('Spanner', () => {
     assert.match(output, /VenueId: 42, VenueName: Venue 42, LastUpdateTime:/);
   });
 
-  // query_with_numeric_parameter
-  it('should use a NUMERIC query parameter to query record from the Venues example table', async () => {
+  // add_numeric_column
+  it('should add a Revenue column to Venues example table', async () => {
     const output = execSync(
-      `${datatypesCmd} queryWithNumeric ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`
+      `${datatypesCmd} addNumericColumn "${INSTANCE_ID}" "${DATABASE_ID}" ${PROJECT_ID}`
+    );
+
+    assert.match(
```

Comment:
If you don't need a regex here, you can use `assert.include`:
```js
assert.include(output, `Waiting for ...`);
```

Reply:
Thanks, that's much neater.